modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -124,14 +119,74 @@ OVN-Kubernetes fully supports the Kubernetes `NetworkPolicy` API in the `network
 [id="how-the-migration-process-works_{context}"]
 == How the migration process works
 
-The migration process works as follows:
+The migration process is summarized in the following table, segmented between the user initiated steps in the process and the actions that the migration performs in response.
+
+[cols="1,1a",options="header"]
+|===
+
+|User initiated|Migration activity
+
+|
+Set the `migration` field of the `Network.operator.openshift.io` custom resource (CR) named `cluster` to `OVNKubernetes`. Make sure the `migration` field is `null` before setting it to a value.
+|
+Cluster Network Operator (CNO):: Updates the status of the `Network.config.openshift.io` CR named `cluster` accordingly.
+Machine Config Operator (MCO):: Rolls out an update to the systemd configuration necessary for OVN-Kubernetes; The MCO updates a single machine per pool at a time by default, so the total time the migration takes increases with the size of the cluster.
+
+|Update the `networkType` field of the `Network.config.openshift.io` CR.
+|
+CNO:: The following actions are performed:
++
+--
+* Destroys the OpenShift SDN control plane pods.
+* Deploys the OVN-Kubernetes control plane pods.
+* Updates the Multus objects to reflect the new cluster network provider.
+--
+
+|
+Reboot each node in the cluster.
+
+Because the existing pods in the cluster are unaware of the change to the Cluster Network Provider, rebooting each node ensures that each node is drained of pods. New pods are attached to the new cluster network provided by OVN-Kubernetes.
+|
+TODO - What k8s mechanism actually drains the node?
```

Comment:
Shall we leave it as a "TODO" here?

Reply:
@pliurh, do you know what cluster process or Operator actually performs the draining when a reboot is triggered? Thanks!

Reply:
The description is not accurate. The node will not be drained when a manual reboot is triggered by the user. That is why we reboot all the nodes at the same time. We don't want the pods to be recreated, instead, we only need the interface of each pod to be created. After node reboot, when the kubelet resumes the pods, it will call the new CNI plugin for each pod.

So we should say 'rebooting each node ensures that the network interface of each pod is recreated. The pods are attached to the new cluster network provided by OVN-Kubernetes.'